test_pull_requests

```diff
@@ -359,6 +361,78 @@ def compose_bot_comment(prefix, comment_data={})
   updated_comment
 end
 
+class FlushableStore
+  def initialize(c_file=nil)
+    @cache_file = nil
+    @cache = {}
+    if c_file
+      @cache_file = File.expand_path(c_file)
+      if File.exists? @cache_file
+        if File.writable? @cache_file
+          begin
+            @cache = Marshal.load IO.read(cache_file)
+          rescue
+            $stderr.puts "  *** Starting with empty cache file at #{c_file}"
+          end
+        else
+          $stderr.puts "  *** Specified cache file #{@cache_file} is not writable. Cache won't persist between runs."
+          @cache_file = nil
+        end
+      else
+        begin
+          FileUtils.touch @cache_file
+        rescue
+          $stderr.puts "  *** Can't create fresh cache file at #{@cache_file}. Cache won't persist between runs."
+          @cache_file = nil
+        end
+      end
+    end
+  end
+
+  def read(key)
```

Comment:
Following three methods are of dubious value -- if this is a simple data storage object, can't we get the internal cache to be accessable by whoever holds the wrapper and not need pass-through methods like this?